docs/index.md

```diff
@@ -0,0 +1,79 @@
+# Welcome to mesa-frames 🚀
+
+mesa-frames is an extension of the [mesa](https://github.com/projectmesa/mesa) framework, designed for complex simulations with thousands of agents. By storing agents in a DataFrame, mesa-frames significantly enhances the performance and scalability of mesa, while maintaining a similar syntax.
+
+## Why DataFrames? 📊
+
+DataFrames are optimized for simultaneous operations through [SIMD processing](https://en.wikipedia.org/wiki/Single_instruction,_multiple_data). Currently, mesa-frames supports two main libraries:
+
+- [Pandas](https://pandas.pydata.org/): A popular data-manipulation Python library, known for its ease of use and high performance.
```

Comment:
The official name is actually pandas, without capitalization, similar to how you'd say iOS instead of IOS when it is at the beginning of the sentence.

Reply:
I see! Let me correct that.